code-challenges.test.js

```diff
@@ -34,13 +100,75 @@ const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
 
 // b) Create the function that makes the test pass.
 
+  // Pseudo coding:
+  // input:arr
+  // output: returns an array of only the REMAINDERS of the numbers when divided by 3.
+  // process: create a function name: reMain3
+  // create a variable: filArr
+  // in filArr use .filter method on arr, declear filter using value, and use typeof(value) method is equal to "number" to get value thats a number.
+  // then return .map method on filArr, declear map using value, and have the value % 3
+  // pass test
+
+  // const reMain3 = (arr) => {
+  //   const filArr = arr.filter((value) => typeof(value) === "number")
+  //   return filArr.map((value) => value % 3)
+  // }
+
+  // pass test:
+  // PASS  ./code-challenges.test.js
+  // reMain3
+  //   ✓ returns an array of only the REMAINDERS of the numbers when divided by 3. (2 ms)
+
 // --------------------3) Create a function that takes in an array of numbers and returns the sum of all the numbers cubed.
 
 // a) Create a test with an expect statement using the variables provided.
 
+  describe("sumCubed", () => {
+    it("returns the sum of all the numbers cubed.", () => {
+      expect(sumCubed(cubeAndSum1)).toEqual(99)
+      expect(sumCubed(cubeAndSum2)).toEqual(1125)
+    })
+  })
+
+  // good fail:
+  // FAIL  ./code-challenges.test.js
+  // sumCubed
+  //   ✕ returns the sum of all the numbers cubed. (1 ms)
+
+  // ● sumCubed › returns the sum of all the numbers cubed.
+
+  //   ReferenceError: sumCubed is not defined
+
+
 const cubeAndSum1 = [2, 3, 4]
 // Expected output: 99
 const cubeAndSum2 = [0, 5, 10]
 // Expected output: 1125
 
 // b) Create the function that makes the test pass.
+
+  // Pseudo coding:
+  // input:arr
+  // output: returns the sum of all the numbers cubed.
+  // process: create a function name: sumCubed
+  // create a variable newArr:
+  // use .map method on arr, declaer using value, have value  * it self 2 times to get new arr with all number cubed value
+  // create another variable but use let to allow it change, sum = 0
+  // use for loop, starting at 0, ending at arr.length, iteration i++
+  // using addtion assignmet with sum and newArr[i]
+  // return sum
+  // pass test
+
+  const sumCubed = (arr) => {
+    const newArr = arr.map((value) => value * value * value)
+    let sum = 0
+    for (let i = 0; i < arr.length; i++ ){
+    sum += newArr[i]
```

Comment:
Watch your indentation! Here's a challenge for you: how can you get rid of the for loop entirely and only use `sum` and `.map` to achieve the solution?